### PR TITLE
Add notice when selecting CDN using CloudFront

### PR DESCRIPTION
**Pull request by cssjoe**

### CODE REVIEW

**Review by mavas84 — DISMISSED** [review]
Exactly what is needed. I'll add the link for the article about the manual purge to avoid invalidations per path

File: Generic_Plugin_Admin.php [hunk 1]
```diff
@@ -439,6 +439,31 @@ function admin_print_scripts() {
...
+							'<br />',
+							'<a target="_blank" href="https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Invalidation.html#PayingForInvalidation">',
+							'</a>',
+						),
+						array(
+							'a' => array(
+								'target' => array(),
+								'href'   => array(),
+							),
+						)
```
Comment (jacobd91) [inline Generic_Plugin_Admin.php:468]: For wp_kses should the <br /> tag be defined as an allowed tag?

File: Generic_Plugin_Admin.php [hunk 2]
```diff
@@ -439,6 +439,31 @@ function admin_print_scripts() {
...
+						sprintf(
+							// translators: 1: HTML break, 2: HTML anchor open tag, 3: HTML anchor close tag.
+							__(
+								'Please see Amazon\'s documentation: Paying for file invalidation%1$sThe first 1,000 invalidation paths that you submit per month are free; you pay for each invalidation path over 1,000 in a month.',
+								'w3-total-cache'
+							),
+							'<br />',
+							'<a target="_blank" href="https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Invalidation.html#PayingForInvalidation">',
+							'</a>',
+						),
```
Comment (jacobd91) [inline Generic_Plugin_Admin.php:461]: For the sprintf, the argument count doesn't match the defined placeholders. Currently only 1 placeholder defined but 3 passed

**Review by jacobd91 — CHANGES_REQUESTED** [review]

**Review by jacobd91 — APPROVED** [review]
Reviewed and approved

**Pull request merged**